autodetect-dependabot.jl

```diff
@@ -0,0 +1,29 @@
+module AutodetectDependabot
+
+function _get_possible_branch_names()
+    possible_branch_names = [
+        get(ENV, "GITHUB_BASE_REF", ""),
+        get(ENV, "GITHUB_HEAD_REF", ""),
+        get(ENV, "GITHUB_REF", ""),
+    ]
+    return possible_branch_names
+end
+
+function _chop_refs_head(branch_name::AbstractString)
+    if startswith(branch_name, "refs/heads/")
+        return replace(branch_name, r"^(refs\/heads\/)" => "")
+    else
+        return branch_name
+    end
```

Comment:
```suggestion
    replace(branch_name, r"^(refs\/heads\/)" => "")
```

If the regex doesn't match anything, nothing will be replaced. We don't need to check for that case manually.